roles/workaround_foreman_remote_execution_core/tasks/main.yml

```
@@ -0,0 +1,20 @@
+---
+- name: Update all packages on the system
+  include_role:
+    name: update_os_packages
+
+- name: Remove rubygem-foreman_remote_execution_core
+  package:
+    state: absent
+    name: rubygem-foreman_remote_execution_core
+  when:
+    - ansible_os_family == "RedHat"
+    - ansible_distribution_major_version == '8'
+
+- name: Remove rubygem-foreman_remote_execution_core
+  package:
+    state: absent
+    name: tfm-rubygem-foreman_remote_execution_core
+  when:
+    - ansible_os_family == "RedHat"
+    - ansible_distribution_major_version == '7'
```

Can we capture this in a single statement?
```suggestion
- name: Remove (tfm-)rubygem-foreman_remote_execution_core
  package:
    state: absent
    name:
      - rubygem-foreman_remote_execution_core
      - tfm-rubygem-foreman_remote_execution_core
  when:
    - ansible_os_family == "RedHat"
```

---

This should be possible, yes

---

We could, I'd like to stick with my explicit version as this work around will be dropped in a week or so.